src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Dowloads a stores Theme.
+   *
+   * @param uuid valid store theme UUID
+   * @returns Promsie resolving to a single theme
+   */
+  get(uuid: number): ThemeAction['GetResponse'] {
+    return this.client.post(`${themesPath}/${uuid}/actions/download`);
+  }
+
+  /**
+   * Activates a Theme
+   *
+   * @param data Data used to activate a Theme
+   * @returns Promise resolving to a newly created customer
```

Comment:
Looks like copy/paste from customers, should update to reflect what is returned from this endpoint

Reply:
Updated to theme info